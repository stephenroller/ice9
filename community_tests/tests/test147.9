# proc overriding global type								succeed

type foo = bool

var b : foo

proc bar (b2 : foo)
	type foo = int
	
	var i : foo
	
	i := 7;
	write i+1;
end

b := true;
bar(b);
write ?b;
