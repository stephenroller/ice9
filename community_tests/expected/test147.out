Loading file: generated/test147.tm
8 
1 
Number of instructions executed = 62
